tools/Voron.Recovery/Recovery.cs

```diff
@@ -486,18 +556,23 @@ void PrintRecoveryProgress(long startOffset, byte* mem, byte* eof, DateTime now)
                             var message =
                                 $"Unexpected exception at position {GetFilePosition(startOffset, mem)}:{Environment.NewLine} {e}";
                             mem = PrintErrorAndAdvanceMem(message, mem);
+                            try
+                            {
+                                recoveredTool.Log(message, e);
+                            }
+                            catch (Exception)
+                            {
```

Comment:
What scenario would cause it to throw here?

Reply:
Some exception in the creation of the log document itself